### ISSUE TITLE
Not change the account picture.﻿

### ISSUE THREAD

**Issue by williamnerdy** [ozonos/bug-tracker-BETA#4]
You can not change the profile image using an image that is in the system.
![screenshot from 2015-03-10 08 32 24](https://cloud.githubusercontent.com/assets/6504353/6574118/0ee10602-c700-11e4-88cf-87a505feb5ed.png)

**Comment by UriHerrera** [comment]
Is that the whole window frame?.

**Comment by williamnerdy** [comment]
Yes, after selecting the picture she gets this size and cannot be resized.

**Comment by williamnerdy** [comment]
I did a test and it worked after disabling the GTK theme.

**Comment by UriHerrera** [comment]
So it doesn't happen with any other theme just Ozon?.

**Comment by williamnerdy** [comment]
That's right, I tested with the following themes:
* Advaita
* HightContrast
* Zukitwo
and not have problems

**Comment by williamnerdy** [comment]
Didn't mean to click on close issue, excuse me!

**Comment by UriHerrera** [comment]
@satya164 might want to check this out.

**Comment by satya164** [comment]
Now, that's interesting.

**Comment by abraaorosa** [comment]
The same problem persists here.

**Comment by loctauxphilippe** [comment]
I have the same issue on my Ozon as well

**Comment by williamnerdy** [comment]
After updating the GTK theme you can change the image.
![screenshot from 2015-04-26 16 12 33](https://cloud.githubusercontent.com/assets/6504353/7338853/a9f7e4a4-ec2f-11e4-9a3b-4ab139f55a54.png)
